### ISSUE TITLE
[Package]: tgpt

### ISSUE THREAD

**Issue by xplshn** [termux/termux-packages#17614]
### Why is it worth to add this package?

This program allows you to use ChatGPT from the terminal, it has history and context support, it works really well and its made in Go, its lightweight and can be statically compiled. No need for API keys or other BS

### Home page URL

https://github.com/aandrew-me/tgpt

### Source code URL

https://github.com/aandrew-me/tgpt

### Packaging policy acknowledgement

- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied in case of violation.

**Comment by landfillbaby** [comment]
you will get banned from openai for using this.

**Comment by xplshn** [comment]
> you will get banned from openai for using this.

Not really, It doesn't need to connect to OpenAI or any other AI really, you don't even need to provide API keys, it uses the API provided by a website called churchless, so, the prgarm isn't the one at fault and also, its just an API, the guy who provides the API never said whether it was from OpenAI or a Foss alternative.

**Comment by xplshn** [comment]
Its not OpenAI, so they can't pin point you and ban you IP.

**Comment by xplshn** [comment]
Also, it isn't exactly OpenAI, its GPT4Free, so it might be using you.com, or forefront ai or etc, there ale various services out there using GPT 3.5

**Comment by TomJo2000** [comment]
The package is still violating both [OpenAI's End User License Agreement](https://openai.com/policies/terms-of-use#:~:text=We%20may%20terminate%20these%20Terms%20immediately%20upon%20notice%20to%20you%20if%20you%20materially%20breach).
And the [Termux packaging policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy:~:text=is%20distributed%20under%20End%20User%20License%20Agreement).

**Comment by xplshn** [comment]
Oh okay, thank you for pointing out it wasnt compliant with Temux's policy, sorry for the trouble.
